phpBB/includes/acp/acp_profile.php

```diff
@@ -672,7 +672,14 @@ function main($id, $mode)
 				$s_one_need_edit = true;
 			}
 
-			$profile_field = $this->type_collection[$row['field_type']];
+			try
+			{
+				$profile_field = $this->type_collection[$row['field_type']];
+			}
+			catch (\Exception $e)
+			{
+				continue;
+			}
```

Comment:
Question: is it better to use the construct as it is now, or use an if(!isset()) instead?  It is more common the use of isset than the try/catch, so maybe better changing it... while I add a blank line.